src/EventStore.Client/HttpFallback.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EventStore.Client {
+	internal class HttpFallback : IDisposable {
+		private readonly HttpClient _httpClient;
+		private readonly JsonSerializerOptions _jsonSettings;
+		private readonly UserCredentials? _defaultCredentials;
+		private readonly string _addressScheme;
+
+		internal HttpFallback (EventStoreClientSettings settings) {
+			var handler = new HttpClientHandler();
+			if (!settings.ConnectivitySettings.Insecure && !settings.ConnectivitySettings.TlsVerifyCert) {
+				handler = new HttpClientHandler() {
```

Comment:
in this case we've constructed a HttpClientHandler a few lines earlier and wont dispose it

probably we can just set these two properties on the one we already constructed?